hhs_oauth_server/settings/base.py

```diff
@@ -517,13 +518,31 @@
 SOCIAL_AUTH_FIELDS_STORED_IN_SESSION = ['next']
 SOCIAL_AUTH_ALWAYS_ASSOCIATE = True
 
+
+MEDICARE_LOGON = True
+MEDICARE_LOGIN_TEMPLATE_NAME = env(
```

Comment:
These URLs shouldn't be configurable. I think you took these out from the code, but they're still here. Can we remove this from here as well?